### PR TITLE
Update the github-codeowner-subscriber version

### PR DESCRIPTION
**Pull request by sima-zhu**
Testing pipeline:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1366353&view=logs&s=7e5a62c6-a3ae-5561-387a-bc0babb014af
Timeout testing:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1369477&view=results
We replace kusto table with opensource API.

### CODE REVIEW

**Review by chidozieononiwu — APPROVED** [review]

**Comment by weshaggard** [comment]
@sima-zhu please be sure to do a test run of the notification pipeline.

**Comment by sima-zhu** [comment]
> @sima-zhu please be sure to do a test run of the notification pipeline.

The test pipeline is in the description.

File: eng/pipelines/notifications.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ stages:
 
   jobs:
   - job: Run
+    timeoutInMinutes: 120
```
Comment (weshaggard) [inline eng/pipelines/notifications.yml:14]: Why did we need to extend the timeout?

File: eng/pipelines/notifications.yml [hunk 2]
```diff
@@ -76,23 +77,13 @@ stages:
             --organization $(Organization) `
             --project $(Project) `
             --path-prefix "\$(PathPrefix)" `
-            --dev-ops-token-var DEVOPS_TOKEN `
```
Comment (weshaggard) [inline eng/pipelines/notifications.yml:79]: After this change gets merged lets be sure to clean-up and remove these secrets from the variable group we are consuming for this pipeline.

File: eng/pipelines/notifications.yml [hunk 3]
```diff
@@ -76,23 +77,13 @@ stages:
...
-            --aad-app-id-var APP_ID `
-            --aad-app-secret-var APP_SECRET `
-            --aad-tenant-var AAD_TENANT `
-            --kusto-url-var KUSTO_URL `
-            --kusto-database-var KUSTO_DB `
-            --kusto-table-var KUSTO_TABLE `
+            --dev-ops-token-var $(azure-sdk-notification-tools-pat) `
+            --aad-app-id-var $(opensource-aad-app-id) `
```
Comment (weshaggard) [inline eng/pipelines/notifications.yml:81]: Looks like you still called these command line parameters with "-var" but you are not passing a variable and instead are passing the value. We should probably follow-up with an update to fix that. We should decide if they should be variables or values.

File: eng/pipelines/notifications.yml (hunk 1 above)
Comment (sima-zhu) [reply eng/pipelines/notifications.yml:14]: Based on recent history, we use more time on AzureDevOpsService call, which made some of the run timeout.

Comment (sima-zhu) [reply eng/pipelines/notifications.yml:14]: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1355827&view=results

**Review by weshaggard — APPROVED** [review]

File: eng/pipelines/notifications.yml (hunk 3 above)
Comment (sima-zhu) [reply eng/pipelines/notifications.yml:81]: I thought it means the value is from variable in pipeline. Make sense to me :)

Comment (sima-zhu) [reply eng/pipelines/notifications.yml:81]: Put it in issue: https://github.com/Azure/azure-sdk-tools/issues/2745

File: eng/pipelines/notifications.yml (hunk 2 above)
Comment (sima-zhu) [reply eng/pipelines/notifications.yml:79]: Put it in issue: https://github.com/Azure/azure-sdk-tools/issues/2745

**Pull request merged**